service/dps/storm/io/src/test/java/eu/europeana/cloud/service/dps/storm/io/HarvestingWriteRecordBoltTest.java

```diff
@@ -272,10 +270,10 @@ public void shouldFailWithNoRetriesIfProviderDoestExistWhenCreatingNewCloudId()
 
 
     private void assertExecutionResults() {
-        verify(outputCollector, times(1)).emit(captor.capture());
+        verify(outputCollector, times(1)).emit(Mockito.any(Tuple.class), captor.capture());
```

Comment:
Here Mockito. can be omitted? any(Tuple.class) should be enough like in the other tests